presto-testng-services/src/main/java/io/prestosql/testng/services/ReportUnannotatedMethods.java

```diff
@@ -77,17 +76,33 @@ private static boolean isTestMethod(Method method)
         if (method.getDeclaringClass() == Object.class) {
             return true;
         }
-        Class<?> superclass = method.getDeclaringClass().getSuperclass();
-        Method overridden;
+        if (method.getDeclaringClass().isInterface()) {
+            return isTemptoClass(method.getDeclaringClass());
```

Comment:
you should not exit early if this is an interface, but not tempto related

```suggestion
        if (method.getDeclaringClass().isInterface() && isTemptoClass(method.getDeclaringClass())) {
            return true;
```